aws/components/sqs/setup.ftl

```diff
@@ -89,6 +93,10 @@
                 solution.DeadLetterQueue.MaxReceives,
                 solution.DeadLetterQueue.MaxReceives > 0,
                 (environmentObject.Operations.DeadLetterQueue.MaxReceives)!3)
+            tags=
+                getOccurrenceCoreTags(
+                    occurrence,
+                    formatComponentFullName(core.Tier, core.Component, sqsName))
```

Comment:
```suggestion
                    sqsName
```

Same as above I think those two values end up being the same

Reply:
Cool, done